Makefile

```diff
@@ -26,9 +26,32 @@ clean_fixtures:
 # extra dep for PostgreSQL targets in pgxs.mk
 clean: clean_fixtures
 
+# Docker stuff
+DOCKER_COMPOSE_COMMAND_BASE=docker-compose --project-directory . --env-file hosting/environment.env
```

Comment:
Looks like `docker-compose` v1 is [already deprecated](https://docs.docker.com/compose/compose-file/). Maybe we could start using v2 here?

Reply:
I approve in theory, but I'd have to upgrade my OS for that.  Which I hope to do at some point, but not today.